presto-main/src/main/java/com/facebook/presto/sql/SqlEnvironmentConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql;
+
+import io.airlift.configuration.Config;
+
+public class SqlEnvironmentConfig
+{
+    //TODO: determine an actual default path
+    private String path = " ";
```

Comment:
Would there be an issue having the default path as a space? Maybe better to keep it empty or null.

Reply:
It can't be null because it's passed to constructors that have requireNonNull(). Otherwise, it's not utilized - it'll be changed once we actually have code that uses the path for function resolution.